src/Sulu/Bundle/MediaBundle/DependencyInjection/SuluMediaExtension.php

```diff
@@ -56,6 +60,22 @@ public function prepend(ContainerBuilder $container)
                 ]
             );
         }
+
+        if ($container->hasExtension('fos_rest')) {
+            $container->prependExtensionConfig(
+                'fos_rest',
+                [
+                    'exception' => [
+                        'codes' => [
+                            MediaNotFoundException::class => 404,
+                            FileVersionNotFoundException::class => 404,
+                            FormatNotFoundException::class => 404,
+                            FormatOptionsMissingParameterException::class => 400,
```

Comment:
Would it make sense to use the `MissingParameterExceptin`, which already exists?